src/browser/api/service.ts

```diff
@@ -32,15 +33,15 @@ export module Service {
         };
     }
 
-    export function getServiceByUuid(uuid: string): ServiceIdentity {
-        return serviceMap.get(uuid);
+    export function getServiceByChannelId(channelId: string): ServiceIdentity {
+        return serviceMap.get(channelId);
     }
 
     // Could be any identifier
-    export function getServiceByName(name: string): ServiceIdentity {
+    export function getServiceByUuid(uuid: string): ServiceIdentity {
         let serviceIdentity;
         serviceMap.forEach((value, key) => {
-            if (value.serviceName === name) {
+            if (value.uuid === uuid) {
                 serviceIdentity = serviceMap.get(key);
```

Comment:
Isn't the result of this `.get(..)` call just the same as `value` to start with?

Consider:

```
serviceMap.forEach(service => {
  if (service.uuid === uuid) {
    serviceIdentity = service;
  }
});
```

or (possibly) better, standard loop instead of `forEach` to allow for short-circuit:

```
for(let serviceIdentity of serviceMap.values()) {
  if (serviceIdentity.uuid === uuid) {
    return serviceIdentity;
  }
}
```